google/cloud/aiplatform/training_jobs.py

```diff
@@ -1602,10 +1603,10 @@ def run(
         """
 
         if self._is_waiting_to_run():
-            raise RuntimeError("Custom Training is already scheduled to run.")
+            raise RuntimeError("AutoML Tabular Training is already scheduled to run.")
```

Comment:
Out of scope for this PR, but in cases like this it might be helpful if the error message includes some additional information such as the training job's full resource name.

Reply:
Captured in [b/177077413](http://b/177077413)